website/content/docs/configuration/session-recording/storage-providers/configure-s3-compliant.mdx

```diff
@@ -0,0 +1,105 @@
+---
+layout: docs
+page_title: Configure S3-compliant storage
+description: |-
+  How to configure an S3-compliant storage provider for Boundary session recording.
+---
+
+# Configure an S3-compliant storage provider
+
+<EnterpriseAlert product="boundary">This feature requires <a href="https://www.hashicorp.com/products/boundary">HCP Boundary or Boundary Enterprise</a></EnterpriseAlert>
+
+The [MinIO plugin](https://github.com/hashicorp/boundary-plugin-minio/) lets you configure S3-compliant storage providers for session recording.
+
+HashiCorp has tested and confirmed that you can configure the following S3-compliant storage products for session recording using the MinIO plugin:
+
+- [Hitachi Content Platform](#hitachi-content-platform-configuration)
+
+It should also be possible to configure other S3-compliant storage providers to work for session recording storage, but we have not tested other providers.
```

Comment:
IMHO we should encourage users to try out other S3 compliant providers. Might want to change this up a little along the lines of:

> Other S3 compliant object storage providers can also be configured for session recording storage and we will add to the list of tested providers as we get around to testing other providers.